provisioning/roles/ruby/tasks/main.yml

```diff
@@ -1,7 +1,31 @@
-- name: ensure Ruby packages are installed
-  apt: pkg={{ item }} state=installed
-  become: yes
-  with_items:
-    - ruby
-    - ruby-dev
-    - rubygems-integration
+---
+- name: install rbenv
+  git:
+    repo: https://github.com/rbenv/rbenv.git
+    dest: ~/.rbenv
+    version: v1.1.0
+
+- name: try to compile dynamic bash extension to speed up rbenv
+  shell: cd ~/.rbenv && src/configure && make -C src
```

Comment:
Please add a [`creates` option](http://docs.ansible.com/ansible/shell_module.html) to avoid running this on every provisioning.